packages/airnode-starter/README.md

```diff
@@ -0,0 +1,40 @@
+# Airnode starter
```

Comment:
Does this need to be a package in the `packages/` directory? What if we created a separate `examples/` directory in the root where we would place various different kinds of examples.

This is what Next.js does and I quite like the way you can find a specific example for whatever the user's specific use case is: https://github.com/vercel/next.js/tree/canary/examples

Reply:
Yeah, that is very nice and I got inspiration for the "each integration in separate folder" from Vercel.

I don't think `examples/` folder works for us though. With Vercel examples, you just clone/download the example you want and run `yarn && yarn dev` and they are self contained. We have to maintain some code (the scripts) for the examples to work...

Another problem is that for the future we would like to run a basic Airnode flow from the starter package as e2e test. Also, it makes it easy to depend on the latest packages in the monorepo this way (which we probably could not do if it wasn't a package).

Reply:
Ok I'm happy with that then